bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: register enterprise cluster

## Description
feat: register enterprise cluster

## Verification Steps
run the code and hit http://localhost:8000/api/kafkas_mgmt/v1/clusters (POST) endpoint. make sure that valid cluster id (alphanumeric with 32 chars length), external cluster id (in the correct format) and dns name (also in the correct format) are provided in the request. Cluster should be registered and in `accepted` state.

Then cluster manager should progress the cluster through various states up until `waiting_for_kas_fleetshard_operator`

tests will follow after initial code review is done

## Checklist (Definition of Done)
